### PR TITLE
[BUGFIX] Handle MySQL 5.7 and above issues with default values

### PR DESCRIPTION
**Pull request by PhilippKuhlmay**
I added some default values for the columns.
I also reordered ext_tables.sql so the additions for the TYPO3 tables are at the top and then all the custom tables.

### CODE REVIEW

**Comment by PhilippKuhlmay** [comment]
This will fix Issues with MySQL 5.7 and above.
#196

File: ext_tables.sql [hunk 1]
```diff
@@ -59,7 +68,7 @@ CREATE TABLE tx_t3blog_com_nl (
     name tinytext NOT NULL,
     post_uid int(11) DEFAULT '0' NOT NULL,
     lastsent int(11) DEFAULT '0' NOT NULL,
-    code tinytext NOT NULL,
+    code varchar(20) DEFAULT '' NOT NULL,
```
Comment (fnagel) [inline ext_tables.sql:71]: Are you sure 20 chars are enough? We create the code with a 32 char max, see https://github.com/fnagel/t3extblog/blob/master/Classes/Domain/Model/AbstractSubscriber.php#L155

File: ext_tables.sql [hunk 2]
```diff
@@ -70,7 +79,7 @@ CREATE TABLE tx_t3blog_com_nl (
 CREATE TABLE tx_t3blog_blog_nl (
     email tinytext NOT NULL,
     lastsent int(11) DEFAULT '0' NOT NULL,
-    code tinytext NOT NULL,
+    code varchar(20) DEFAULT '' NOT NULL,
```
Comment (fnagel) [inline ext_tables.sql:82]: see above

File: ext_tables.sql [hunk 3]
```diff
@@ -1,3 +1,12 @@
+#
+# Table structure for table 'tt_content'
+#
+CREATE TABLE tt_content (
+    irre_parentid int(11) DEFAULT '0' NOT NULL,
+    irre_parenttable varchar(20) DEFAULT '' NOT NULL
```
Comment (fnagel) [inline ext_tables.sql:6]: Here, 20 chars should be good.

File: ext_tables.sql (hunk 1 above)
Comment (PhilippKuhlmay) [reply ext_tables.sql]: I didn't see the max size of the char in AbstractSubscriber.php
I increased the value to 32

File: ext_tables.sql (hunk 2 above)
Comment (PhilippKuhlmay) [reply ext_tables.sql]: see above

**Pull request merged**